tests/Operation/DeleteTest.php

```diff
@@ -65,4 +65,30 @@ public function provideInvalidConstructorOptions()
 
         return $options;
     }
+
+    public function testExplainableCommandDocument(): void
+    {
+        $options = [
+            'collation' => ['locale' => 'fr'],
+            'hint' => '_id_',
+            // Ignored options
+            'let' => ['a' => 1],
+            'ordered' => true,
+            'comment' => 'explain me',
```

Comment:
`ordered` shouldn't be included here, as it's not documented as an option for Delete.

As for `let` and `comment`, those should probably appear in the generated command document as top-level fields. Quoting [Explain: Command Fields](https://www.mongodb.com/docs/manual/reference/command/explain/#command-fields):

> If you specify explain without a comment, it inherits any comment in the command specified to explain.

So, the `comment` option is certainly relevant to include. I'm not sure about `let`, but I don't think there's any harm in including that as well. You can also hunt down someone from the server query team if you want a definitive answer on that.

Also, I recall that `writeConcern` was removed in https://github.com/mongodb/mongo-php-library/commit/118f08409fec7ce8be8664dc7bea0803842f8d2c for [PHPLIB-802](https://jira.mongodb.org/browse/PHPLIB-802). For operations where you explicitly excluding `writeConcern` from `Explainable::getCommandDocument()`, I think it'd make sense to include it in `$options` with a comment noting that it is intentionally ignored.